src/components/Plans/Checkout/PaymentMethod/Transfer/Colombia/index.test.js

```diff
@@ -0,0 +1,99 @@
+import { render, screen } from '@testing-library/react';
+import IntlProvider from '../../../../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { AppServicesProvider } from '../../../../../../services/pure-di';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom/extend-expect';
+import { actionPage } from '../../../Checkout';
+import { Formik, Form } from 'formik';
+import { TransferColombia } from '.';
+import { act } from 'react-dom/test-utils';
+import { getFormInitialValues } from '../../../../../../utils';
+import { fieldNames } from '../../PaymentMethod';
+
+const initialPropsReonlyView = {
+  readOnly: true,
+};
+
+const initialPropsUpdateView = {
+  readOnly: false,
+};
+
+const TransferColombiaElement = ({ paymentMethod, optionView }) => {
+  return (
+    <AppServicesProvider>
+      <IntlProvider>
+        <BrowserRouter>
+          <Formik initialValues={getFormInitialValues(fieldNames)}>
+            <Form>
+              {optionView === actionPage.UPDATE ? (
+                <TransferColombia {...initialPropsUpdateView} paymentMethod={paymentMethod} />
+              ) : (
+                <TransferColombia {...initialPropsReonlyView} paymentMethod={paymentMethod} />
+              )}
+            </Form>
+          </Formik>
+        </BrowserRouter>
+      </IntlProvider>
+    </AppServicesProvider>
+  );
+};
+
+describe('Transer Colombia component', () => {
+  it('should show the correct data in readonly view', async () => {
+    //Arrange
+    const paymentMethod = {
+      identificationNumber: '123456',
+      razonSocial: 'Test Company',
+    };
+
+    // Act
+    await act(async () => {
```

Comment:
Why act & async ? you check if it's necessary or not: [Guide lines - tests](https://github.com/FromDoppler/doppler-webapp#test)

Reply:
I removed this